src/store/student-store.js

```diff
@@ -53,6 +53,15 @@ class StudentStore extends Store {
         return this.getStudentById(student.id) === undefined;
     }
 
+    isUniqueRfid(rfid){
+        let returnVal = true;
+        this.getStudents().forEach(student => {
```

Comment:
Nit: make this fail-fast instead of having to visit every student: `Array.prototype.some`